### ISSUE TITLE
a question about hyperparameter settings

### ISSUE THREAD

**Issue by shangxinyi** [VICO-UoE/DatasetCondensation#8]
Hi, thanks for the great work.  I try to follow the experimental setting and generate 1 condensed image per class for CIFAR10 with ResNet18BN_AP without data augmentation. However, the test accuracy is only 16%. How should I set the hyperparameters for the ResNet network?
`Hyper-parameters: 
{method: DC, dataset: CIFAR10, model: ResNet18BN_AP, ipc: 1, eval_mode: S, num_exp: 5, num_eval: 20, epoch_eval_train: 300, Iteration: 1000, lr_img: 0.1, lr_net: 0.01, batch_real: 256, batch_train: 256, init: noise, dsa_strategy: None, data_path: data/CIFAR10, save_path: result, dis_metric: ours, outer_loop: 1, inner_loop: 1, device: cuda, dsa: False}`

**Comment by PatrickZH** [comment]
Hi, thanks for the question. You used correct hyperparameters. Learning synthetic images with ResNet on CIFAR10 is still challenging for DC method (ICLR 2021). If you use the synthetic set learned with ResNet18BN_AP to train ConvNet, you should obtain a higher accuracy around 26%. The possible reason is that sophisticated architectures like ResNet need more data to train the parameters, thus they may perform worse than simpler architectures. Please refer to the Cross-architecture Generalization Experiments in https://arxiv.org/pdf/2110.04181.pdf for more details about cross-architecture experiments on CIFAR10.

**Comment by shangxinyi** [comment]
您好，我是尚心怡，我已收到您的邮件，会尽快给您回复，辛苦了！
